packages/paste-core/components/datagrid/src/DataGridContext.tsx

```diff
@@ -0,0 +1,14 @@
+import * as React from 'react';
+import type {CompositeStateReturn} from '@twilio-paste/reakit-library';
+
+interface DataGridState extends Partial<CompositeStateReturn> {
+  actionable: boolean;
+  setActionable: (value: React.SetStateAction<boolean>) => void;
+}
+
+const NOOP: () => void = () => {};
```

Comment:
```suggestion
const NOOP: VoidFunction = () => {};
```

Would make this easier to read.

Reply:
Cool! TIL